exampleSite/config.toml

```diff
@@ -1,4 +1,5 @@
 # Override these settings with your own
+title = "codex"
```

Comment:
this overrides changes made in https://github.com/jakewies/hugo-theme-codex/pull/65.

My thoughts are that we should try to keep everything as consistent as possible. What's your thoughts here?

Reply:
Oh I just want to add a default value for `title`, it'll just appear in the `# Override these settings` block.

Reply:
Oh I see. Quickly saw this and thought it was:

```
theme = "codex"
```

Whoops! No need to rebase!

Reply:
oh haha 👍